### ISSUE TITLE
Getting post information directly

### ISSUE THREAD

**Issue by hackmoore** [davidteather/TikTok-Api#61]
Assuming we have the id of a post we want to check, is there a way anyone knows that we can use to get more information about an individual post directly rather than querying the user and trawling through their posts to match the id. This would save on potentially a lot of additional requests.

Thanks!

**Comment by davidteather** [comment]
This api is kind of a mess right now and I don't think any of the methods actually are working. They fixed their APIs weaknesses for now and selenium is being detected.

**Comment by hackmoore** [comment]
Yeah I have been keeping track, I meant more about an endpoint similar to `https://m.tiktok.com/share/item/list?secUid=&id=%s&type=1&count=24&minCursor=0&maxCursor=0&shareUid=&lang=`

which just returns a single item - should have been more clear - I'm working on something semi-independent

**Comment by davidteather** [comment]
In a future version I'll attempt to add this, but right now my signature generating method only works for the /api/item_list/ path. If you've got a solution to this I'd be happy to add something like that

**Comment by davidteather** [comment]
I should be able to implement this in a few days I’ve been busy recently. I’ll also be adding a method to return the tiktok object using a url.
